mobile/app/src/main/java/org/exarhteam/iitc_mobile/IITC_WebView.java

```diff
@@ -56,9 +56,9 @@ private void iitc_init(final Context c) {
         mSettings.setDomStorageEnabled(true);
         mSettings.setAllowFileAccess(true);
         mSettings.setGeolocationEnabled(true);
-        mSettings.setAppCacheEnabled(true);
+      //  mSettings.setAppCacheEnabled(true);
```

Comment:
Is this an intentional change?

Reply:
No maybe i tryed something... dont remember